packages/opencensus-core/test/test-console-exporter.ts

```diff
@@ -60,11 +44,15 @@ describe('NoopExporter', () => {
     it('should do anything', () => {
       const exporter = new NoopExporter();
       const rootSpan = new RootSpan(tracer);
-      const queue: RootSpan[] = [];
-      queue.push(rootSpan);
+      const queue: RootSpan[] = [rootSpan];
 
-      exporter.publish(queue);
-      assert.ok(true);
+      exporter.publish(queue)
+          .then(() => {
+            assert.ok(true);
+          })
+          .catch((err: Error) => {
+            assert.fail(err.message);
```

Comment:
Instead of using `.catch` and failing in the catch handler, just return `exporter.publish(queue)`.

Reply:
Way better this way. Just changed, thanks :D